src/content/docs/infrastructure/host-integrations/host-integrations-list/lighttpd-integration.mdx

```diff
@@ -68,7 +68,7 @@ server.modules = (
     "mod_status"
```

Comment:
Kindly remove 1 from "mod_1indexfile". It would be "mod_indexfile" .

Reply:
done!